### PR TITLE
Updated explainer

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR updates the introduction of the Web NFC explainer by adding user flow.

### CODE REVIEW

**Review by zolkis — APPROVED** [review]

File: EXPLAINER.md [hunk 1]
```diff
@@ -10,19 +10,27 @@ Participate
...
 of the API the focus has been on supporting the NFC Data Exchange Format aka
 [NDEF](https://www.oreilly.com/library/view/beginning-nfc/9781449324094/ch04.html),
 a lightweight binary message format, as it works across different tag formats.
 
-The focus has been on making an easy to use, and very webby API that web
-developers would be comfortable with, and at the same time making sure that it
-is low-level and flexible enough so that users can get the full potential out of
-NDEF and even integrate with legacy solutions.
+Web NFC is only available to top-level, secure browsing contexts, and origins
+must first request the “nfc” permission while handling a user gesture. To then
```
Comment (kenchris) [inline EXPLAINER.md:23]: remove then

File: EXPLAINER.md [hunk 2]
```diff
@@ -10,19 +10,27 @@ Participate
...
+perform a read or write, the web page must be visible when the user touches
```
Comment (kenchris) [inline EXPLAINER.md]: you dont "touch to"

when the user touches an NFC tag with their device - or brings an NFC tag within reading range of their device..

File: EXPLAINER.md [hunk 3]
```diff
@@ -10,19 +10,27 @@ Participate
...
+their device to an NFC tag. The browser uses haptic feedback to indicate a tap.
```
Comment (kenchris) [inline EXPLAINER.md]: a tap = a tag of some kind was read

File: EXPLAINER.md [hunk 4]
```diff
@@ -10,19 +10,27 @@ Participate
...
+Access to the NFC radio is blocked if the display is off or the device is
+locked. For backgrounded web pages, receiving and pushing NFC content are
+suspended.
+
+We believe Web NFC is an easy to use, and very webby API that web developers
```
Comment (kenchris) [inline EXPLAINER.md]: webby? an API that fits well with the API design of the web platform?

File: EXPLAINER.md (hunk 1 above)
Comment (beaufortfrancois) [reply EXPLAINER.md:23]: "then" indicate the action happens after the "nfc" permission was granted. I think we should keep it for this.

File: EXPLAINER.md (hunk 2 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: Fixed.

File: EXPLAINER.md (hunk 4 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: I've removed "webby" as we already mention that "web developers will be comfortable with"

File: EXPLAINER.md (hunk 3 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: a tap is pretty universal to understand.

File: EXPLAINER.md (hunk 1 above)
Comment (kenchris) [reply EXPLAINER.md:23]: And at that point...

File: EXPLAINER.md (hunk 3 above)
Comment (kenchris) [reply EXPLAINER.md]: ok

**Pull request merged**